myhpom/models/__init__.py

```diff
@@ -1,4 +1,8 @@
 from django.db import models
+from .state import State
+from .user import User
+from .user_details import UserDetails
+
 
 AD_PACKET_STATES = (
```

Comment:
I like that this PR migrates the structure from app/models.py to app/models/a_model.py - for consistency, pull the model out of here into its own file?

Reply:
Now that I'm looking at how State/StateAdvanceDirective are related, it seems to me that this model should go away, and that the `advance_directive_template` should replace `State.supported`. I like the idea of having a follow on ticket to address this and the flow of the whole 'user signup process': make sure a user is signed in, that the duplication of these two models is addressed, the 'next steps' URL pattern no longer includes the State, and that the user lands on a dashboard page with a working logout button (so that testers can repeat tests easily). Does that sound like a reasonable follow-on ticket? If so...disregard the refactor suggestion above!

Reply:
(defer to MH-58) State.supported is currently being used for sorting purposes, so if that column is replaced the sorting logic will also need to be addressed.